wm/desk.go

```diff
@@ -192,14 +199,125 @@ func (x *x11WM) configureWindow(win xproto.Window, ev xproto.ConfigureRequestEve
 	}
 }
 
+func (x *x11WM) clientMessage(ev xproto.ClientMessageEvent) {
```

Comment:
This is a big method with many indented switch statements that should probably be broken out to separate functions. Also "clientMessage" sounds like an action but it's really processing an event no?